query-engine/prisma-models/tests/datamodel_converter_tests.rs

```diff
@@ -38,31 +38,125 @@ fn converting_enums() {
     }
 }
 
+// region: composite
 #[test]
-fn converting_composite_types() {
+fn converting_composite_types_compound() {
     let res = psl::parse_schema(
         r#"
-    datasource db {
-      provider = "mongodb"
-      url      = "mongodb://localhost:27017/hello"
-    }
+        datasource db {
+            provider = "mongodb"
+            url      = "mongodb://localhost:27017/hello"
+        }
 
-    model MyModel {
-      id        String      @id @default(auto()) @map("_id") @db.ObjectId
-      attribute Attribute
+        model Post {
+            id         String      @id @default(auto()) @map("_id") @db.ObjectId
+            author     User        @relation(fields: [authorId], references: [id])
+            authorId   String      @db.ObjectId
+            attributes Attribute[]
+      
+            @@unique([authorId, attributes])
+            //       ^^^^^^^^^^^^^^^^^^^^^^
+            // Prisma does not currently support composite types in compound indices
```

Comment:
Where does this shortened error msg come from?

Reply:
You commented on it above; it's from the `composite_type_in_compound_index` function in `validations/indexes.rs`

Reply:
But doesn't that continue with `Please remove {:?} from the index.`? Where does that half of the message go?

Reply:
It was just supposed to be a highlight of where the message would be in the schema :)